lib/postgrex/pgpass.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Postgrex.Pgpass do
+  @moduledoc """
+  Parses .pgpass file to obtain database credentials
+  """
+
+  @pgpass_path Application.get_env(:postgrex, :pgpass, Path.join(System.user_home, '.pgpass'))
```

Comment:
Very good point. Based on @fishcakez comment above, I'm gonna ditch the module attribute.